application: twitter-portablecontacts
version: 1
threadsafe: false
api_version: 1
default_expiration: 1d

# ideally i'd use python27, but i develop with ubuntu lucid's python 2.6, and
# its httplib.HTTPResponse is incompatible. tweepy.auth.get_authorization_url()
# complains that HTTPResponse.readline doesn't exist:
#
# File ".../portablecontacts-unofficial/tweepy/auth.py", line 75, in _get_request_token
#   resp = urlopen(Request(url, headers=request.to_header()))
# File ".../google_appengine/google/appengine/dist27/urllib.py", line 926, in __init__
#   self.readline = self.fp.readline
runtime: python25

handlers:
- url: /static
  static_dir: static

- url: /(.well-known/.*)?
  script: app.py
  secure: always

- url: /(start_auth|oauth_callback)
  script: twitter_demo.py
  secure: always

- url: /poco(/.*)?
  script: poco.py
  secure: always
